src/script/interpreter.cpp

```diff
@@ -1707,6 +1711,9 @@ bool GenericTransactionSignatureChecker<T>::CheckSchnorrSignature(Span<const uns
 
     uint8_t hashtype = SIGHASH_DEFAULT;
     if (sig.size() == 65) {
+        if (m_require_sighash_all) {
```

Comment:
Although `SIGHASH_DEFAULT` implies `SIGHASH_ALL`, AFAIK using `SIGHASH_ALL` is still possible for users who wish to waste 1 byte. So I think you need `m_require_sighash_all && nHashType != SIGHASH_ALL` here too.

Reply:
Isn't that a malleability? Anyway, gotcha.